docs/debugging.md

```diff
@@ -14,97 +14,46 @@ React Native provides an in-app developer menu which offers several debugging op
 
 Alternatively for Android devices and emulators, you can run `adb shell input keyevent 82` in your terminal.
 
-![The React Native Dev Menu](/docs/assets/debugging-dev-menu.jpg)
+![The React Native Dev Menu](/docs/assets/debugging-dev-menu-076.jpg)
 
 :::note
 The Dev Menu is disabled in release (production) builds.
 :::
 
-## Opening the Debugger
+## Opening DevTools
 
-The debugger allows you to understand and debug how your JavaScript code is running, similar to a web browser.
+[React Native DevTools](./react-native-devtools) is our included debugger for React Native. It allows you to inspect and understand how your JavaScript code is running, similar to a web browser.
 
-:::info
-**In Expo projects**, press <kbd>j</kbd> in the CLI to directly open the Hermes Debugger.
-:::
-
-<Tabs groupId="js-debugger" queryString defaultValue={constants.defaultJsDebugger} values={constants.jsDebuggers}>
-<TabItem value="hermes">
-
-Hermes supports the Chrome debugger by implementing the Chrome DevTools Protocol. This means Chrome's tools can be used to directly debug JavaScript running on Hermes, on an emulator or on a physical device.
-
-1. In a Chrome browser window, navigate to `chrome://inspect`.
-2. Use the "Configure..." button to add the dev server address (typically `localhost:8081`).
-3. You should now see a "Hermes React Native" target with an **"inspect"** link. Click this to open the debugger.
-
-![Overview of Chrome's inspect interface and a connected Hermes debugger window](/docs/assets/debugging-hermes-debugger-instructions.jpg)
-
-</TabItem>
-<TabItem value="flipper">
-
-[Flipper](https://fbflipper.com/) is a native debugging tool which provides JavaScript debugging capabilities for React Native via an embedded Chrome DevTools panel.
-
-To debug JavaScript code in Flipper, select **"Open Debugger"** from the Dev Menu. Learn more about Flipper [here](https://fbflipper.com/docs/features/react-native/).
-
-:::info
-To debug using Flipper, the Flipper app must be [installed on your system](https://fbflipper.com/docs/getting-started/).
-:::
-
-![The Flipper desktop app opened to the Hermes debugger panel](/docs/assets/debugging-flipper-console.jpg)
+To open DevTools:
```

Comment:
```suggestion
To open React Native DevTools you can do one of these two:
```

I think it's better to use the full name.

Reply:
Becomes repetitive (which I've also intentionally done at the leading end of sections or in primary statements). I'd like us to be able to refer to "DevTools" alone, which is also done in parts of the React Native (Dev Menu, launch flow) and Chrome frontend UIs.

Reply:
It's ok, but then we need to specify that we will call the `React Native DevTools` simply  `DevTools` for sake of brevity. Otherwise a reader might start thinking that the `React Native DevTools` and the `DevTools` are two different things.

Especially because, at some point, you talk about the `React DevTools`, with no `Native`.

Reply:
What I've just noticed for sure is we can simplify out one repetition later in this paragraph 😅

> including the integrated React ~~DevTools~~ Components Inspector and Profiler